cmd/age/age.go

```diff
@@ -174,19 +174,23 @@ func main() {
 		defer f.Close()
 		out = f
 	} else if terminal.IsTerminal(int(os.Stdout.Fd())) {
-		if armorFlag {
-			// If the output will go to a TTY, and it will be armored, buffer it
-			// up so it doesn't get in the way of typing the input.
+		if !armorFlag && name != "-" {
+			if decryptFlag {
+				// TODO: buffer the output and check it's printable.
+			} else {
```

Comment:
```
		if name != "-" {
			if decryptFlag {
				// TODO: buffer the output and check it's printable.
			} else if !armorFlag {
```

`decryptFlag` implies `!armorFlag` so this is the same, but it makes the decision tree clearer.

Reply:
I agree, this looks clearer. I will update the pull request.